cmd/servicelog/post.go

```diff
@@ -87,14 +89,39 @@ func (o *PostCmdOptions) Validate() error {
 	return nil
 }
 
+func (o *PostCmdOptions) CheckServiceLogsLastHour() bool {
+	getAllMessages := false      // we need just manual entries
+	getInternalLogsOnly := false // we need all messages
+	numberOfHours := 1           // number of hours we need to wait for svc logs
+	response, err := FetchServiceLogs(o.ClusterId, getAllMessages, getInternalLogsOnly)
+	if err != nil {
+		log.Fatalf("failed to fetch service logs: %q", err)
+	}
+	timeStampToCompare := time.Now().Add(-time.Hour * time.Duration(numberOfHours))
+	for _, svclog := range response.Items().Slice() {
+		timestamp := svclog.Timestamp()
+		if timestamp.After(timeStampToCompare.UTC()) {
+			return true // prompt message for re-confirm
+		}
+	}
+	return false
+}
+
```

Comment:
In case there was at least one service log, we probably want to print it/them as it is necessary information for the decision whether a new SL should be sent or not.